addons/mail/models/bus_listener_mixin.py

```diff
@@ -17,5 +17,10 @@ def _bus_send_store(
             store = args[0]
         else:
             store = Store(*args, **kwargs)
+        if not subchannel:
+            subchannel = self._default_bus_subchannel()
```

Comment:
I don't like the idea of a "default" sub-channel, as this is security sensitive having a non-safe (internal) by default is prone to issue.

The default should be the "unsafe" one always, which should be the one when nothing is provided.

I'd prefer if the channel was always explicitly given I think.